packages/api-types/types/api.d.ts

```diff
@@ -4449,17 +4449,17 @@ export interface components {
       default_pool_size: number | null
       max_client_conn: number | null
       /** @enum {string|null} */
-      pool_mode: 'transaction' | 'session' | 'statement' | null
```

Comment:
I don't think `null` is possible for `pool_mode`.

Reply:
btw we shouldn't update this file - this is auto generated from the API, which means this needs to be fixed on the API instead

Reply:
Gotcha, thanks

Reply:
ahh sorry - i assumed you did this manually 🤦 just to check if you ran npm run codegen in packages/api-types?

Reply:
i didn't, wasn't sure how it worked.

Reply:
thanks for doing it! now i know